lib/internal/errors.js

```diff
@@ -393,6 +378,16 @@ function uvErrmapGet(name) {
   return uvBinding.errmap.get(name);
 }
 
+function captureLargerStackTrace(err) {
+  userStackTraceLimit = Error.stackTraceLimit;
+  Error.stackTraceLimit = 20;
```

Comment:
Why `20` is chosen as the trace limit? I don't see such restriction in the old code base.

Reply:
It is an arbitrary limit, yes
There is no easy way to determine in advance how many additional calls will be there
Unless you successively try larger and larger sizes

Reply:
If there is no good reason behind the constant, I'd suggest to remove the limit here. Otherwise, this PR may introduce a regression in user applications' exception handling. We're not in control of the user code and it may have arbitrary call depth and/or `Error.stackTraceLimit` value.